src/TEntityResolver.cpp

```diff
@@ -0,0 +1,134 @@
+/***************************************************************************
+ *   Copyright (C) 2020 by Gustavo Sousa - gustavocms@gmail.com            *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+#include "TEntityResolver.h"
+#include <QDebug>
+
+QString TEntityResolver::getResolution(const QString& entity) const
+{
+    if (entity.front() != '&' || entity.back() != ';')
```

Comment:
Could you still wrap single line ifs with braces - we've adopted that codestyle across the codebase. Apple's Goto fail was enlightening here.

Reply:
> Could you still wrap single line ifs with braces - we've adopted that codestyle across the codebase. Apple's Goto fail was enlightening here.

Ok, I'll ty to stick to it. What do you use for helping to enforce the coding style?

Is there any naming convention you use in the project for classes. I am putting these Ts in front of class names just because they are already used around, but I didn  get what is the meaning.

Reply:
`clang-format` - we have a `.clang-format` file at the root of the repo with the style. Unfortunately clang-format doesn't handle adding braces - there was some other tool that does, but we keep it simple with 1 formatter.

Naming convention - for variables it's `m` for a member variable, `s` for static and `c` for constant, hence, `scm` in a few places. I'm not sure about classes, to be honest :O

Reply:
And a `p` for pointer.

*\*quoting many a teenage contestant on the UK ITV Quiz show from the late 80s "BlockBusters" to the host Bob Holness [as in this Celebratory reprisal hosted by Dara O'Brean](https://www.youtube.com/watch?v=BS5WUZ3k68g)\**:

"Can I have a P(ee) please Bob."